docs/Parameters-Tuning.rst

```diff
@@ -81,4 +81,6 @@ Deal with Over-fitting
 
 -  Try ``extra_trees``
 
+-  Try increasing ``path_smoothing``
```

Comment:
We should either add `path_smoothing` into param's aliases or fix name here to ``path_smooth`. Otherwise users will not be able to find this term anywhere in our docs.